pre_commit/commands/hook_impl.py

```diff
@@ -157,6 +159,11 @@ def _run_ns(
         return _ns(hook_type, color, commit_msg_filename=args[0])
     elif hook_type in {'pre-merge-commit', 'pre-commit'}:
         return _ns(hook_type, color)
+    elif hook_type == 'post-checkout':
+        return _ns(
+            hook_type, color, origin=args[0], source=args[1],
```

Comment:
origin and source are actually the opposite direction, the forward diff is `ORIGIN...SOURCE`